### PR TITLE
Add API docs link to About card

### PR DESCRIPTION
**Pull request by canut**
## Hey, I just made a Pull Request!

This PR implements the feature request described in https://github.com/spotify/backstage/issues/3010. I used the best icon I could find to represent API docs. Let me know if you prefer another one.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/spotify/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

Icon enable
image

Icon disable
image

### CODE REVIEW

File: plugins/catalog/src/components/AboutCard/AboutCard.tsx [hunk 1]
```diff
@@ -120,6 +121,14 @@ export function AboutCard({ entity, variant }: AboutCardProps) {
                 entity.metadata.namespace || ENTITY_DEFAULT_NAMESPACE
               }/${entity.kind}/${entity.metadata.name}`}
             />
+            <IconLinkVertical
+              disabled={!entity.spec?.implementsApis}
+              label="View API"
+              icon={<BrightnessAutoIcon />}
+              href={`./${
```
Comment (stefanalund) [inline plugins/catalog/src/components/AboutCard/AboutCard.tsx:128]: @freben does this look correct?

**Review by stefanalund — COMMENTED** [review]
Thanks! Let's use the same image as we do for the API-docs plugin:

https://material.io/resources/icons/?icon=extension&style=baseline

File: plugins/catalog/src/components/AboutCard/AboutCard.tsx (hunk 1 above)
Comment (freben) [reply plugins/catalog/src/components/AboutCard/AboutCard.tsx:128]: I thought this would be just `./api` right?

Comment (canut) [reply plugins/catalog/src/components/AboutCard/AboutCard.tsx:128]: I thought the same. It was my first implementation but it was wrong.

Comment (freben) [reply plugins/catalog/src/components/AboutCard/AboutCard.tsx:128]: No I verified that this is actually correct, but I fail to understand why react-router thinks it's necessary. :( We could merge for now and then debug what's going on under the hood

**Review by freben — APPROVED** [review]

File: plugins/catalog/src/components/AboutCard/AboutCard.tsx (hunk 1 above)
Comment (Rugvip) [reply plugins/catalog/src/components/AboutCard/AboutCard.tsx:128]: This also assumes that [this](https://github.com/spotify/backstage/blob/e11865ec69cb76bf03fcb415a51fc808c08b4337/packages/app/src/components/catalog/EntityPage.tsx#L179) route isn't modified in the app.

Comment (freben) [reply plugins/catalog/src/components/AboutCard/AboutCard.tsx:128]: fixed in #3037

**Pull request merged**